src/OmniSharp/Api/v1/ProjectSystem/OmnisharpController.ProjectSystem.cs

```diff
@@ -24,11 +25,15 @@ public class ProjectSystemController
 
         [HttpPost("/projects")]
         [HttpGet("/projects")]
-        public WorkspaceInformationResponse ProjectInformation()
+        public WorkspaceInformationResponse ProjectInformation(ProjectInformationRequest request)
         {
+            var includeSourceFiles = true;
+            if (request != null)
```

Comment:
If clients are already making use of this action method and they do not send any parameters it will throw an NRE (NullReferenceException). Are you ok with that ?